BrainPortal/app/helpers/rich_ui_helper.rb

```diff
@@ -46,6 +46,12 @@ def overlay_description(description="", options={})
     link.html_safe
   end
 
+  # Takes a +description+ with multiple lines and shows
+  # the whole thing, including line breaks.
+  def full_description(description)
+    ("<span style=\"white-space: pre-wrap;\">" + description + "</span>").html_safe
```

Comment:
this is wrong; special characters entered in the description will now be interpreted by the browser. Anyone can execute arbitrary code in the browser by entering a description with javscript statements, or just plainly break the page with spruious html tag